databases/login_db.txt

```diff
@@ -1,3 +1,3 @@
-00000001,LukeMaher,CarlFrohburg,LennartMantel,SamuelFoley,testUser,
-00000001,17223563,17237823,17207266,17237874,00000000,
-user,user,user,user,user,admin,
\ No newline at end of file
+l,t,
+l,t,
```

Comment:
Get rid of your changes, just checkout the file from the main branch

Reply:
Gonna have to explain that one

Reply:
Well, I assume we want to keep our names in there and IDs. You wana do "git checkout origin main -- databases/login_db.txt"